simulator/src/main/java/com/hazelcast/simulator/worker/tasks/IMultipleProbesWorker.java

```diff
@@ -1,45 +0,0 @@
-/*
- * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.hazelcast.simulator.worker.tasks;
-
-import com.hazelcast.simulator.probes.Probe;
-
-import java.util.Map;
-import java.util.Set;
-
-/**
- * Interface for {@link IWorker} implementations, which should get individual {@link Probe} instances per operation injected.
- */
-public interface IMultipleProbesWorker extends IWorker {
```

Comment:
Not needed anymore since the Worker can get access to the PropertyBinding and fish the probes out.